docs/en/Mounting-Alluxio-FS-with-FUSE.md

```diff
@@ -1,104 +1,100 @@
 
-### Optional configuration steps
-
-Alluxio-FUSE is based on the standard java `alluxio-core-client-fs` to perform its operations. You
-might want to customize the behaviour of the alluxio client used by Alluxio-FUSE the same way you
-would for any other client application.
-
-One possibility, for example, is to edit `$ALLUXIO_HOME/integration/fuse/bin/alluxio-fuse.sh` and add your specific alluxio client options in the `ALLUXIO_JAVA_OPTS` variable.
-
-## Operational assumptions and status
+### Unmount Alluxio-FUSE
 
-Currently, most basic file system operations are supported. However, due to Alluxio implicit characteristics, please, be aware that:
+To umount a previously mounted Alluxio-FUSE file sytem, on the node where the file system is mounted, point a shell to your `$ALLUXIO_HOME` and run:
 
-* Files can be written only once, only sequentially, and never be modified.
-* Due to the above, any further access to a file must be read-only.
+```bash
+$ integration/fuse/bin/alluxio-fuse umount mount_point
+```
 
-This translates in the following constraints on the UNIX system calls that will operate on the file system:
+This unmounts the file system at the mounting point and stops the corresponding alluxio-fuse process. For example,
 
-### `open(const char* pathname, int flags, mode_t mode)` (see also `man 2 open`)
+```bash
+$ integration/fuse/bin/alluxio-fuse umount /mnt/people
+Unmount fuse at /mnt/people (PID: 97626).
+```
 
-If `pathname` indicates the path of a non-existing regular file in Alluxio, then an open will only succeed if:
+### Check the Alluxio-FUSE mounting status
 
-1. The base directory of `pathname` exists in Alluxio;
-2. `O_CREAT` and `O_WRONLY` are passed among the `flags` bitfield.
+To list the mounting points, on the node where the file system is mounted, point a shell to your `$ALLUXIO_HOME` and run:
 
-Equivalently, `creat(const char* pathname )` calls will succeed as long as (1) holds and `pathname` does not exist yet.
+```bash
+$ integration/fuse/bin/alluxio-fuse stat
+```
 
-If `pathname`, instead, points to an existing regular file in Alluxio, then an open call will only succeed if:
+This outputs the `pid, mount_point, alluxio_path` of all the running alluxio-fuse processes.
 
-1. `O_RDONLY` is passed among the `flags` bitfield.
+For example, the output will be like:
 
-Note that, in either cases, the `mode` parameter is currently ignored by Alluxio-FUSE.
+```bash
+pid	mount_point	alluxio_path
+80846	/mnt/people	/people
+80847	/mnt/sales	/sales
+```
 
-### `read(int fd, void* buf, size_t count)` (see also `man 2 read`)
+## Optional configuration steps
 
-A read system call will only succeed when `fd` refers to an Alluxio file that has been previously opened with the `O_RDONLY` flags.
+Alluxio-FUSE is based on the standard Java client API `alluxio-core-client-fs` to perform its operations. You
```

Comment:
`Alluxio-FUSE is based on the standard Java client API `alluxio-core-client-fs` to perform its operations` I don't think it is necessary to expose this level of details to users in this tutorial

Reply:
This is to help user understand why changing alluxio client properties would affect alluxio fuse